### PR TITLE
Bump version for branch main to 1.7

### PR DESCRIPTION
**Pull request by ReToCode**
Bumping versions for SO branch main to 1.7

### CODE REVIEW

**Comment by nak3** [comment]
Hmm... As per commit history, this version was upgraded by bot https://github.com/openshift/release/commit/1876dffa9633ba435168b94d357f546ca4d6d76f So, we probably missed something :thinking:

File: ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.11-aws-ovn.yaml [hunk 1]
```diff
@@ -7,20 +7,20 @@ base_images:
...
     tag: cli
-  openshift_eventing-kafka-broker-src_knative-v1.6:
+  openshift_eventing-kafka-broker-src_knative-v1.7:
     name: eventing-kafka-broker-src
     namespace: openshift
-    tag: knative-v1.6
-  openshift_knative-eventing-src_knative-v1.6:
+    tag: knative-v1.7
+  openshift_knative-eventing-src_knative-v1.7:
     name: knative-eventing-src
```
Comment (nak3) [inline ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.11-aws-ovn.yaml:15]: It seems `knative-eventing-src` was replaced with `eventing-src` by this commit https://github.com/openshift/release/commit/9e7f88f58d31608b63ff62933ec840431423c28c#diff-904a63d931dc06dc6f80feaeb5fd08bec7a9e22bf75eb81c652b34851fada744L204

@matzew @pierDipi Could you give your advice what we should change :sweat_drops:

Comment (matzew) [reply ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.11-aws-ovn.yaml:15]: I think these images are not yet on the registry at all

**Pull request closed without merging**